### PR TITLE
DSN-15 Registration and login page

### PR DESCRIPTION
**Pull request by patvoj**

### CODE REVIEW

File: DSN/frontend/src/app/login/index.html
Comment (bela-riboczki-ext) [inline DSN/frontend/src/app/login/index.html:1]: Can you create this as a react page?
We are losing a lot of features when we have baked in scripts and we just have this as a simple html. Let the next.js combine the elements together.

https://nextjs.org/docs/pages/building-your-application/routing/pages-and-layouts

File: DSN/frontend/src/app/signup/index.html
Comment (bela-riboczki-ext) [inline DSN/frontend/src/app/signup/index.html:1]: Can you create this as a react page?
We are losing a lot of features when we have baked in scripts and we just have this as a simple html. Let the next.js combine the elements together.

https://nextjs.org/docs/pages/building-your-application/routing/pages-and-layouts

**Review by bela-riboczki-ext — CHANGES_REQUESTED** [review]
It's seems there is a lot of rename changes in this pull request. Is it possible that the latest code from develop was not merged? Are these valid renames of the back end files?

Please check my individual comments.

**Comment by patvoj** [comment]
It should be working now. There is one feature I didn't manage to implement and it's auto redirect after successful signup/login. Redirection works with route.push and I have trouble to make this method works.

**Review by bela-riboczki-ext — APPROVED** [review]
Looks good from my side.

**Review by pintermate225 — APPROVED** [review]
I see no issues, good from my side too

**Pull request merged**